RobotFramework_Testsuites/Config/CConfig.py

```diff
@@ -549,19 +549,18 @@ def __loadConfigFileLevel2(self):
         except Exception as error:
             CConfig.bLoadedCfg = False
             CConfig.sLoadedCfgError = str(error)
-            logger.error("Loading of JSON configuration file failed! Reason: %s" %(CConfig.sLoadedCfgError))
+            logger.error(f"Loading of JSON configuration file failed! Reason: %s" %(CConfig.sLoadedCfgError))
```

Comment:
Hi Son,
please study this here: https://realpython.com/python-f-strings/
Since around 1 year the rule is to use everywhere f-strings where variables are need to be used in strings.
f-strings are better to read and less error prone than

Correct would be:
```logger.error(f"Loading of JSON configuration file failed! Reason: '{CConfig.sLoadedCfgError}'")```

Please take care in future,
Thank you,
Thomas

Reply:
merged, but please use f-strings as per my example.